homeassistant/components/camera/__init__.py

```diff
@@ -160,6 +160,14 @@ class Image:
     raise HomeAssistantError("Unable to get image")
 
 
+@bind_hass
+async def async_get_image_url(hass, entity_id):
```

Comment:
We can get the state object and get the "entity_picture" attribute. `hass.states.get(entity.entity_id).attributes["entity_picture"]`

Reply:
😭 I really need to spend time reviewing helpers/utilities, and the hass object... I spent too much time trying to get that attribute since it wasn't included in the `AlexaEntity` class. 

Thanks for this.